source/extensions/listener_managers/listener_manager/listener_impl.cc

```diff
@@ -946,7 +946,14 @@ Init::Manager& PerListenerFactoryContextImpl::initManager() { return listener_im
 
 bool ListenerImpl::createNetworkFilterChain(
     Network::Connection& connection, const Filter::NetworkFilterFactoriesList& filter_factories) {
-  return Configuration::FilterChainUtility::buildFilterChain(connection, filter_factories);
+  if (Configuration::FilterChainUtility::buildFilterChain(connection, filter_factories)) {
```

Comment:
Cleaner to make this `if (!...) { ... return false}` and then return true after.

Reply:
done